MatrixSDK/IdentityServer/MXIdentityService.m

```diff
@@ -254,9 +254,20 @@ - (MXHTTPOperation*)checkAPIVersionAvailabilityWithSuccess:(void (^)(void))succe
     } failure:^(NSError * _Nonnull error) {
         self.restClient.preferredAPIPathPrefix = kMXIdentityAPIPrefixPathV1;
```

Comment:
`self.restClient.preferredAPIPathPrefix` should be set only in case of "success"